src/components/booklist.js

```diff
@@ -2,24 +2,54 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/bookSlice';
+import '../index.css';
 
 const Book = (props) => {
   const { book } = props;
   const dispatch = useDispatch();
   return (
-    <div className="book-container">
+    <div className="books">
       {book.map((item) => (
-        <div key={item.id}>
-          <h3>{item.title}</h3>
-          <h4>{item.author}</h4>
-          <p>{item.category}</p>
-          <span>
-            <button type="button">Comments</button>
-            <button type="button" onClick={() => dispatch(removeBook(item.id))}>
-              Remove
-            </button>
-            <button type="button">Edit</button>
-          </span>
+        <div className="book-content" key={item.id}>
+          <div className="book-card">
+            <div className="book-detail">
+              <strong>{item.category}</strong>
+              <h3>{item.title}</h3>
+              <p>{item.author}</p>
+            </div>
+            <span className="btn-status">
+              <button type="button">Comments</button>
+              <button
+                className="remove-btn"
+                type="button"
+                onClick={() => dispatch(removeBook(item.id))}
+              >
+                Remove
+              </button>
+              <button type="button">Edit</button>
+            </span>
+          </div>
+          <div className="book-progress">
+            <div className="progress">
+              <div className="progress-bar" />
+            </div>
+            <div className="progress-info">
+              <p className="completed">
+                {item.complete}
+                %
+              </p>
+              <span>Completed</span>
+            </div>
+            <div className="progress-status">
+              <p>Current Chapter</p>
+              <span>
+                Chapter
+                {' '}
+                {item.chapter}
+              </span>
+              <button type="button">Update Progress</button>
+            </div>
+          </div>
         </div>
```

Comment:
- [ ] Kindly improve your styling to match the design. Your app should match the design exactly. This requires taking into consideration the font type, size, button appearance, spacing, and so on. Kindly improve the design especially the font and spaces between elements. Kindly compare your styling with the design

- [ ] You can see within the subsection on the right the elements are somewhat aligned to the left, and there is enough space on the right.

  your app | design
  :---:|:---:
  ![image](https://user-images.githubusercontent.com/81468675/222405093-f8a2a898-3079-4507-bcd0-c814b980d050.png) | ![etq](https://user-images.githubusercontent.com/81468675/222405224-f1902365-1519-4c6d-8da6-382f4cab93b8.jpg)